### PR TITLE
Add test for header component

### PR DESCRIPTION
**Pull request by zarinaazamatova**

### CODE REVIEW

File: src/components/Header/components/Bonus/Bonus.test.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { render, screen } from '@testing-library/react';
+import { ThemeProvider } from 'styled-components';
+import defaultTheme from '../../../../styles/theme';
+import { Bonus } from './Bonus';
+
+describe('Bonus', () => {
+  it('test', () => {
```
Comment (Alex-2kZharkov) [inline src/components/Header/components/Bonus/Bonus.test.tsx:7]: give a descriptive description. For example, `should render component without errors`. By the way, the first word is always `should`

File: src/components/Header/components/Catalog/Catalog.test.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import { render, screen } from '@testing-library/react';
+import { ThemeProvider } from 'styled-components';
+import defaultTheme from '../../../../styles/theme';
+import { Catalog } from './Catalog';
+
+jest.mock('../../../../assets/list.png', () => {
+  return { __esModule: true, default: () => <div /> };
+});
+describe('Catalog', () => {
+  it('test', () => {
```
Comment (Alex-2kZharkov) [inline src/components/Header/components/Catalog/Catalog.test.tsx:11]: same

**Review by Alex-2kZharkov — CHANGES_REQUESTED** [review]

File: src/components/Header/components/Bonus/Bonus.test.tsx (hunk 1 above)
Comment (zarinaazamatova) [reply src/components/Header/components/Bonus/Bonus.test.tsx:7]: ✅

File: src/components/Header/components/Catalog/Catalog.test.tsx (hunk 2 above)
Comment (zarinaazamatova) [reply src/components/Header/components/Catalog/Catalog.test.tsx:11]: ✅

**Review by Alex-2kZharkov — APPROVED** [review]

**Pull request merged**